test/integration/graphql_test.lua

```diff
@@ -1639,3 +1639,39 @@ function g.test_schema_input_arg_described_with_kind_variable_pass()
     local _, errors = check_request(query, query_schema, nil, nil, { variables = variables })
     t.assert_equals(errors, nil)
 end
+
+function g.test_arguments_default_values()
```

Comment:
The test succeeds disregarding whether the patch on `schema.lua` is applied or not.

Reply:
Fixed